src/constants.js

```diff
@@ -87,6 +87,11 @@ export const DEFAULT_SERVERS = [
   'https://node2.mainnet.hathor.network/v1a/',
 ];
 
+export const DEFAULT_MINING_SERVERS = {
+  'mainnet': 'https://txmining.mainnet.hathor.network/',
+  'testnet': 'https://txmining.testnet.hathor.network/',
+}
+
```

Comment:
The lib has this information in the `src/config.ts` file. Should we export it from there, so we don't duplicate this information?

Reply:
I think exporting from the lib is the best approach, but until then I don't see why we can't leave this here.